Test/Unit/Helper/DataTest.php

```diff
@@ -124,6 +124,13 @@ public function testGetPspReferenceSearchUrl($expectedResult, $pspReference, $ch
         $this->assertEquals($expectedResult, $pspSearchUrl);
     }
 
+    public function testGetPspReferenceWithNoAdditions(){
+
+        $this->assertEquals(`852621234567890A`, $this->dataHelper->getPspReferenceWithNoAdditions('852621234567890A'));
+        $this->assertEquals(`852621234567890A`, $this->dataHelper->getPspReferenceWithNoAdditions('852621234567890A-refund'));
+        $this->assertEquals(`852621234567890A`, $this->dataHelper->getPspReferenceWithNoAdditions('852621234567890A-capture'));
```

Comment:
```suggestion
        $this->assertEquals(`852621234567890A`, $this->dataHelper->getPspReferenceWithNoAdditions('852621234567890A-capture'));
        $this->assertEquals(`852621234567890A`, $this->dataHelper->getPspReferenceWithNoAdditions('852621234567890A-capture-refund'));
```

Reply:
yes, this is more real world scenario :)